src/main/webapp/scripts/keywords.js

```diff
@@ -54,3 +95,10 @@ $().ready(function() {
     return true;
   });
 });
+
+function addKeyword(key, value, map) {
+  let btn = $('<a></a>').attr('href', '/results?k=' + key);
+  let keywords = value.join(', ');
+  btn.text(keywords);
+  keywordsDiv.append(btn);
+}
```

Comment:
Are you using this function?

Reply:
I forgot to remove this function, thanks for pointing this out!